code/modules/mechs/interface/screen_objects.dm

```diff
@@ -1,5 +1,5 @@
 // Screen objects hereon out.
-#define MECH_UI_STYLE(X) "<span style=\"font-family: 'Small Fonts'; -dm-text-outline: 1 black; font-size: 5px;\">" + X + "</span>"
+#define MECH_UI_STYLE(X) ("<span style=\"font-family: 'Small Fonts'; -dm-text-outline: 1 black; font-size: 5px;\">" + X + "</span>")
```

Comment:
Why do you need this for string???

Reply:
I guess it's not really important for strings... still, it's better to keep standards... standardized.

More importantly, this should probably just be a single string with a [x] inside of it.

Reply:
> More importantly, this should probably just be a single string with a [x] inside of it.

Wait no I'm stupid